backend/spindle/views.py

```diff
@@ -7,51 +8,81 @@
 import base64
 
 
+class SpindleError(Enum):
+    SPINDLE = "spindle"
+    LATEX = "latex"
+    GENERAL = "general"
+
+
+class SpindleResponse:
```

Comment:
Very cool! Perhaps you can transform it into a `dataclass` (from `dataclasses` standard module), so that you don't have to write the constructor.

Reply:
Oh, I did not know about this! Very convenient! 👍